[srp-client] add lease changing test

simonlingoogle opened the pull request:
Verified that the test can be fixed by https://github.com/openthread/openthread/pull/7281

wgtdkp commented on tests/scripts/thread-cert/node.py:
```diff
@@ -1111,6 +1111,16 @@ def srp_client_get_services(self):
         return [self._parse_srp_client_service(line) for line in service_lines]
 
+    def srp_client_set_lease_interval(self, leaseinterval: int):
+        cmd = 'srp client leaseinterval %d' % leaseinterval
```
```suggestion
        cmd = f'srp client leaseinterval {leaseinterval}'
```

simonlingoogle commented:
TIP:
- Need merge https://github.com/openthread/openthread/pull/7281 first.

@wgtdkp @jwhui

abtink reviewed (approved):
LGTM. 👍

wgtdkp commented:
@simonlingoogle #7281 is merged, please rebase！

simonlingoogle commented:
> @simonlingoogle #7281 is merged, please rebase！

Actually, I rebased on #7281 days ago but the CI was still failing. 

I have fixed the CI by changing Wireshark configuration in the 2nd commit.

wgtdkp commented on tests/scripts/thread-cert/pktverify/consts.py:
```diff
@@ -338,6 +338,25 @@
 
 WIRESHARK_DECODE_AS_ENTRIES = {
     'udp.port==61631': 'coap',
+    'udp.port==53536': 'dns',
+    'udp.port==53543': 'dns',
+    'udp.port==53544': 'dns',
+    'udp.port==53538': 'dns',
+    'udp.port==53539': 'dns',
+    'udp.port==53540': 'dns',
+    'udp.port==53541': 'dns',
+    'udp.port==53546': 'dns',
+    'udp.port==53547': 'dns',
+    'udp.port==53548': 'dns',
+    'udp.port==53552': 'dns',
+    'udp.port==53554': 'dns',
+    'udp.port==53542': 'dns',
+    'udp.port==53545': 'dns',
+    'udp.port==53549': 'dns',
+    'udp.port==53551': 'dns',
+    'udp.port==53553': 'dns',
+    'udp.port==53535': 'dns',
+    'udp.port==53537': 'dns',
```
sort the entries?

simonlingoogle replied:
sorted.

wgtdkp reviewed (approved):
LGTM :+1:
